app/helpers/poem_helper.rb

```diff
@@ -1,2 +1,7 @@
 module PoemHelper
+  def markdown(text)
+    renderer = Redcarpet::Render::HTML.new(filter_html: true)
+    markdown = Redcarpet::Markdown.new(renderer, extensions = {})
```

Comment:
__[rubocop]__
* Useless assignment to variable - `extensions`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)